### DISCUSSION TITLE
Demo?

### DISCUSSION THREAD

**Discussion by timmeade** [vercel/next-forge#14] [General]
Would be nice to see a demo.   It looks like an awesome piece of work, but I personally do not want to setup all those other accounts to make it work.    

Any plans for this?

**Comment by SawkaDev** [comment]
+1

**Comment by Ashutosh-Malve** [comment]
+1

**Comment by princegeco** [comment]
+1

**Comment by haydenbleasel** [comment]
Demo’s almost ready, will ship it shortly

**Reply by Ashutosh-Malve** [reply]
Thanks

**Reply by Ashutosh-Malve** [reply]
Lord, @haydenbleasel , we are still waiting for demo 🥹

**Reply by haydenbleasel** [reply]
Still getting the various deploys up but here's what I have so far:

- `web`: https://demo.next-forge.com/
- `api`: https://api.demo.next-forge.com/health
- `app`: https://app.demo.next-forge.com/

**Comment by haydenbleasel** [comment]
Hey folks, we have a fully live demo now! https://docs.next-forge.com/overview#demo
